File: src/main/java/com/waitring/waitring/repository/StoreRepository.java
package com.waitring.waitring.repository;

import com.waitring.waitring.entity.Store;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface StoreRepository extends JpaRepository<Store, Long>, StoreRepositoryCustom {
    List<Store> findByNameContainingOrAreaDongContaining(String name, String areaDong);
}

File: src/main/java/com/waitring/waitring/repository/StoreRepository.kt
package com.waitring.waitring.repository

import com.waitring.waitring.entity.Store
import org.springframework.data.jpa.repository.JpaRepository

interface StoreRepository : JpaRepository<Store, Long>, StoreRepositoryCustom {
    fun findByNameContainingOrAreaDongContaining(name: String, areaDong: String): List<Store>
}